clients/client-python/build.gradle.kts

```diff
@@ -162,16 +162,20 @@ tasks {
   }
 
   val build by registering(VenvTask::class) {
+    dependsOn(pylint)
+    venvExec = "python"
+    args = listOf("scripts/generate_version.py")
   }
 
   val distribution by registering(VenvTask::class) {
+    dependsOn(build)
     doFirst {
       delete("README.md")
       generatePypiProjectHomePage()
       delete("dist")
     }
 
-    venvExec = "Python3"
+    venvExec = "python"
```

Comment:
Why do we change this?

Reply:
I found the test on github actions failed and said "Python3 is not found" when I use same `venvExec` in `build` task. And `distribution` task seems not tested on github action, so maybe it would error originally.